conda_build/main_build.py

```diff
@@ -30,6 +30,7 @@
     'numpy': [16, 17, 18, 19, 110],
     'perl': None,
     'R': None,
+    'lua': ["2.0", "5.1", "5.2", "5.3"]
```

Comment:
I'm not so sure -- `perl` and `R` both allow `.` characters in their versions.  Only python and numpy use the strange `27` and `110` conventions.  Since that doesn't generalize very well (especially when there are more than two version numbers, such as R's `5.18.2`), then IMHO using dots should be the preferred convention.  We should keep the python/numpy version convention around for historical reasons, but just for those two "langs", not for new langs.